sklearn/datasets/species_distributions.py

```diff
@@ -154,23 +154,24 @@ def fetch_species_distributions(data_home=None,
         The latitude/longitude values for the grid are discussed below.
         Missing data is represented by the value -9999.
 
-    train : record array, shape = (1623,)
+    train : record array, shape = (1624,)
         The training points for the data.  Each point has three fields:
 
         - train['species'] is the species name
         - train['dd long'] is the longitude, in degrees
         - train['dd lat'] is the latitude, in degrees
 
-    test : record array, shape = (619,)
+    test : record array, shape = (620,)
         The test points for the data.  Same format as the training data.
 
-    Nx, Ny : integers
+    Nx, Ny : integers, values : Nx = 1212, Ny = 1592
         The number of longitudes (x) and latitudes (y) in the grid
 
     x_left_lower_corner, y_left_lower_corner : floats
+    values : x_left_lower_corner = -94.8, y_left_lower_corner = -56.05
```

Comment:
Unless this is a separate field (in which case it's incorrectly formatted for other reasons), please indent.